app/javascript/admin_entrypoints/charts.ts

```diff
@@ -3,8 +3,4 @@ import Chartkick from 'chartkick';
 
 import 'chartjs-adapter-luxon';
 
-// @ts-expect-error NOTE: Importing Chartkick seems to overwrite `window.$` with
-// Chartkick's `Chart` object, which is minified as `$`? Undo this.
-window.$ = window.jQuery;
```

Comment:
I think we don't need this anymore because we are doing `type="module"` for the scripts now (which I think keeps variables separate from each other within the scripts, even if the variables have the same names).